[IMP] payment_pro: Add payment_difference_handling in payment

maq-adhoc opened the pull request.

maq-adhoc committed:
[IMP] payment_pro: Add payment_difference_handling in payment

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/account-payment/pull/472).

maq-adhoc committed:
[IMP] read add is confirm

maq-adhoc committed:
[IMP] account_payment_pro: re add matched_move_line_ids in payment

maq-adhoc committed:
[IMP] add account_payment_pro_receiptbook module

jjscarafia committed:
[REM] payment_pro_receiptbook: next_number

jjscarafia committed:
[REM] payment_pro_receiptbook: deprecate sequence manual

Vimos que se usaba muy poco y tenia mucha complejidad. De ser necesario podemos evaluar re-implementar pero lo ideal sería que no

jjscarafia committed:
[IMP] payment_pro: receiptbooks install

jjscarafia committed:
[FIX] payment_pro: varios fixes

jjscarafia commented:
@roboadhoc r+ rebase-ff

roboadhoc commented:
Merge method set to rebase and fast-forward.
